marklogic/documents.py

```diff
@@ -272,9 +273,17 @@ def _extract_values_from_header(part) -> dict:
 
     content_type = None
     if part.headers.get("Content-Type".encode(encoding)):
-        content_type = part.headers["Content-Type".encode(encoding)].decode(encoding)
+        content_type = part.headers["Content-Type".encode(encoding)].decode(
```

Comment:
I am thinking the block before this that splits the disposition string on ";" and "=" needs to be removed, as it presumably is not going to work for any URI with those symbols in it. But the caller to this function needs to know about "category" and "version_id" in the returned header values. 

Can you see if the `Message` class provides access to those as well? I am hoping it properly splits out the entire disposition string and then provides access to those attributes. 

In theory, a test should fail for these URIs containing "=" and ";" (include an example URI that has both) if you try to test on "version_id" for it. I don't recall when that value gets populated, but there are multiple tests that verify it's at least not None.